cmd/commands.go

```diff
@@ -0,0 +1,190 @@
+package xcaddycmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/caddyserver/xcaddy"
+	"github.com/caddyserver/xcaddy/internal/utils"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	buildCommand.Flags().StringArray("with", []string{}, "can be used multiple times to add plugins by specifying the Go module name and optionally its version, similar to go get. Module name is required, but specific version and/or local replacement are optional.")
+	buildCommand.Flags().String("output", "", "changes the output file.")
+	buildCommand.Flags().StringArray("replace", []string{}, "is like --with, but does not add a blank import to the code; it only writes a replace directive to go.mod, which is useful when developing on Caddy's dependencies (ones that are not Caddy modules). Try this if you got an error when using --with, like \"cannot find module providing package\".")
+	buildCommand.Flags().StringArray("embed", []string{}, "can be used multiple times to embed directories into the built Caddy executable. The directory can be prefixed with a custom alias and a colon : to use it with the root directive and sub-directive.")
+}
+
+var versionCommand = &cobra.Command{
+	Use:   "version",
+	Long:  "Command to get xcaddy version",
+	Short: "xcaddy version",
```

Comment:
We may be able to eliminate one of them. I'm not sure what will the display be like when each is eliminated.

```suggestion
	Long:  "Prints xcaddy version",
	Short: "Prints xcaddy version",
```

Reply:
Kept short as it's the one that gets displayed for both help commands (at the tool and the command levels)